code/modules/medical/pathology/pathogen_tools.dm

```diff
@@ -60,26 +60,26 @@
 	examine()
 		if (src.dirty || src.dirty_reason)
 			. = ..()
-			. += "<span style=\"color:red\">The petri dish appears to be incapable of growing any pathogen, and must be cleaned.</span>"
+			. += "<span style=\"color:red\">The petri dish appears to be incapable of growing any pathogen, and must be cleaned.</span><br/>"
 			return
-			
-		. = list("This is [src]")
+
+		. = list("This is [src]<br/>")
 		if (src.reagents.reagent_list["pathogen"])
 			var/datum/reagent/blood/pathogen/P = src.reagents.reagent_list["pathogen"]
-			. += "<span style=\"color:blue\">It contains [P.volume] units of harvestable pathogen.</span>"
+			. += "<span style=\"color:blue\">It contains [P.volume] units of harvestable pathogen.</span><br/>"
 		if (src.medium)
-			. += "<span style=\"color:blue\">The petri dish is coated with [src.medium.name].</span>"
-		. += "Nutrients in the dish:"
+			. += "<span style=\"color:blue\">The petri dish is coated with [src.medium.name].</span><br/>"
+		. += "Nutrients in the dish:<br/>"
 		var/count = 0
 		for (var/N in nutrition)
 			if (nutrition[N])
 				if (nutrition[N] != 1)
-					. += "<span style=\"color:blue\">[nutrition[N]] units of [N]</span>"
+					. += "<span style=\"color:blue\">[nutrition[N]] units of [N]</span><br/>"
```

Comment:
recommend that you remove the `if (nutrition[N] != 1)` if clause and make this message `[nutrition[N]] unit\s of [N]`, the \s macro will take care of the s.

Reply:
Thanks, a very good point! I updated the pathogen amount display a bit above to use \s macro as well.